### ISSUE TITLE
Update for Laravel 8.0

### ISSUE THREAD

**Issue by wimil** [felixkiss/uniquewith-validator#117]

**Comment by lloricode** [comment]
resolve #116

**Comment by makroxyz** [comment]
Please rmerge pull request and release.

Thanx

**Comment by schranzer235** [comment]
push 💘

**Comment by EmilMoe** [comment]
Is this project dead?

**Comment by elijahchancey** [comment]
Please merge the laravel 8 support. Thanks!

**Comment by r8928** [comment]
@felixkiss Please merge

**Comment by EmilMoe** [comment]
You  can omit this package:

```php
return [
    'name' => 'string',
    'email' => 'email',
    'partner_id' => Rule::unique('users', 'partner_id')->where(function($query) {
        $query->where('client_id', User::findOrFail($this->user)->client_id)
            ->where('id', '<>', $this->user);
    }),
];
```

**Comment by elijahchancey** [comment]
please do this when you can, thanks!

**Comment by EM-LilianaIturribarria** [comment]
@felixkiss when is this going to be ready?

**Comment by felixkiss** [comment]
Merged #116 
Released in [3.4.1](https://packagist.org/packages/felixkiss/uniquewith-validator#3.4.1)
